Tests/Templating/AbstractFilterTest.php

```diff
@@ -15,9 +15,17 @@
 use Liip\ImagineBundle\Templating\FilterExtension;
 use Liip\ImagineBundle\Templating\Helper\FilterHelper;
 use Liip\ImagineBundle\Tests\AbstractTest;
+use Symfony\Component\Templating\Helper\Helper;
 
 abstract class AbstractFilterTest extends AbstractTest
 {
+    public static function setUpBeforeClass(): void
+    {
+        if (!class_exists(Helper::class)) {
+            static::markTestSkipped('Test requires deprecated symfony/templating component');
+        }
+    }
```

Comment:
`symfony/templating` is deprecated and removed from the framework in 7.0.  So, this can't be tested when working with versions of the framework without the component are installed.